examples/examples_test.sh

```diff
@@ -71,6 +71,7 @@ EXAMPLES=(
     "features/orca"
     "features/retry"
     "features/unix_abstract"
+    "features/gracefulstop"
```

Comment:
This needs expectations for server and/or client output to confirm it is working correctly.

Reply:
Added. PTAL